.gitignore

```diff
@@ -2,6 +2,8 @@
 main
 web/harbour_bridge_output/
 harbour_bridge_output/
```

Comment:
Can we remove these please?

Reply:
Local repos will have old output files. Lets keep it for some time in order to avoid accidentally committing them?